packages/docs/src/routes/docs/cookbook/nav-link/index.mdx

```diff
@@ -0,0 +1,67 @@
+# NavLink Component
+
+When building navigation in Qwik, you may want to know when a link is active or pending navigation. The standard <a> and <Link> components don't provide this. 
+The NavLink component enhances navigation links by adding:
+
+- **Active Status** - Apply a class when the link matches the current location.
+- **Pending Status** - Apply a class during navigation when linking to the target location.
+
+This allows styling the active and pending states for navigation.
+
+## Usage
+
+Using NavLink is the same as Link, with the addition of `isActive` and `isPending` props:
+
+```jsx
+<NavLink href="/" isActive="active" isPending="pending">
+  Home
+</NavLink>
+```
+
+## How NavLink Works 
+
+Under the hood, NavLink uses the useLocation hook to get navigation status. It checks if the link href matches the current URL pathname to set isActive. It uses isNavigating to determine pending status when navigating to the link.
+
+This allows NavLink to know the active and pending state automatically based on navigation.
+
+```tsx
```

Comment:
Can you change this snippet with a component you can check the other examples with this imports
`import CodeSandbox, {CodeFile} from '../../../../components/code-sandbox/index.tsx';`
This helps us ensure alignment over time.

Reply:
sorry i don't understand

Reply:
The other cookbook examples are using `CodeSandbox` and `CodeFile` to ensure alignment over time.
Can you do the same pls?

Reply:
done